### PR TITLE
Status Update 6

### PR DESCRIPTION
**Pull request by degeri**

### CODE REVIEW

File: src/content/en/news/status-update6.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: "Status Update"
+draft: false
+slug: "status-update"
+date: 2020-03-16
+type: "page"
+---
+
+We have processed a total of 104 submissions so far, with 11 of them being eligible for a payout. 
+
+The PR linked below fixes a potential node crash in dcrd 1.4.0 . The network hardforked on (13/03/2020) and these older vulnerable clients have been forked off. Hence this vulnerability has little to no impact on the working of the network.
```
Comment (jholdstock) [inline src/content/en/news/status-update6.md]:
```suggestion
The PR linked below fixes a potential node crash in dcrd 1.4.0. The network hardforked on 13/03/2020 and these older vulnerable clients have been forked off, hence this vulnerability has little to no impact on the working of the network.
```

File: src/content/en/news/status-update6.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+[DCRD Memory exhaustion via unbounded knownAddresses map](https://github.com/decred/dcrd/pull/1683)
```
Comment (jholdstock) [inline src/content/en/news/status-update6.md]:
```suggestion
[dcrd memory exhaustion via unbounded knownAddresses map](https://github.com/decred/dcrd/pull/1683)
```

File: src/content/en/news/status-update6.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+With this update, all security issues identified via the bounty program have been made public.
```
Comment (jholdstock) [inline src/content/en/news/status-update6.md]: How about adding something like this?

> All bug reporters are given the chance to be listed in the hall of fame, and we are happy to welcome Aaron Hook who discovered the issue above. Thanks for participating Aaron!

Comment (degeri) [reply src/content/en/news/status-update6.md]: Sure lets do that.

**Pull request merged**